charts/histogram/src/ext.js

```diff
@@ -19,8 +20,25 @@ export default function ext(env) {
       sharing: true,
       viewData: true,
     },
-    importProperties: null, // Prevent conversion to and from this object
-    exportProperties: null,
+    importProperties: (exportFormat, initialProperties, extension, hypercubePath) => {
+      const dataDefinition = data(env);
+      const layoutService = { layout: { templateId: initialProperties.templateId } };
+      extension.mapProperties(layoutService);
```

Comment:
```suggestion
      extension.mapProperties();
```
no layoutService is required

Reply:
Fixed!